spi/policy-spi/src/main/java/org/eclipse/dataspaceconnector/spi/policy/store/PolicyStore.java

```diff
@@ -60,5 +60,12 @@
      * @throws EdcPersistenceException if something goes wrong.
      */
     @Nullable
-    Policy delete(String policyId);
+    Policy deleteById(String policyId);
+
+    /**
+     * If the store implementation supports caching, this method triggers a cache-reload.
+     */
+    default void reload() {
```

Comment:
I saw this method in a lot of `Store`s, probably could deserve a separate interface, since it's like an additional feature on some stores

Reply:
yeah, the entire caching topic may be one worth exploring in more detail. We could look at having a generic cache implementation, maybe even using exchangeable eviction strategies.